src/encrypt_only.rs

```diff
@@ -0,0 +1,177 @@
+use super::{derive_static_secret, ValidationErr};
+use chacha20poly1305::{
+    aead::{Aead, AeadCore, KeyInit},
+    ChaCha20Poly1305,
+};
+use js_sys::Error;
+use rand_core::OsRng;
+use schnorrkel::SecretKey;
+use serde::{Deserialize, Serialize};
+use serde_json::to_string;
+use sp_core::{sr25519, Bytes};
+use wasm_bindgen::prelude::*;
+use x25519_dalek::PublicKey;
+use zeroize::Zeroize;
+
+#[wasm_bindgen(js_name = encryptOnly)]
+/// Encrypts, signs, and serializes a SignedMessage to JSON.
+pub fn encrypt_only(sk: Vec<u8>, msg: Vec<u8>, pk: Vec<u8>) -> Result<String, Error> {
```

Comment:
this shouldn't have a sk though?

Reply:
The secret key is used to derive an encryption keypair. We could omit this argument and generate an ephemeral one internally.

But i think if doing that, we may as well also sign the message with it and not have to have an EncryptedMessage type.

Personally i think the best solution is to close this PR, and create an ephemeral signing keypair on `entropy-js` when making a request, so that the signature request each node receives can be checked they come from the same source, even if the keypair is only used for that particular message.

Reply:
ya we can do that, should be simple enough and we can use that account ID in the session ID all solutions handled

Reply:
but we would need to make a function that allows for the account to be passed in rather then derived from the sk